scheduler-interface-spec.md

```diff
@@ -270,11 +270,21 @@ message UpdateResponse {
   // Accepted Applications
   repeated AcceptedApplication acceptedApplications = 7;
 
+  // Updated Applications
+  repeated UpdatedApplication updatedApplications = 8;
+
   // Rejected Node Registrations
-  repeated RejectedNode rejectedNodes = 8;
+  repeated RejectedNode rejectedNodes = 9;
 
   // Accepted Node Registrations
-  repeated AcceptedNode acceptedNodes = 9;
+  repeated AcceptedNode acceptedNodes = 10;
+}
+
+message UpdatedApplication {
+  // The application ID that was updated
+  string applicationID = 1;
+  // State of the application
+  string state = 2;
```

Comment:
please add 2 more fields:
1. stateTransitionTimestamp
2. a string message that can help to explain what happened

Reply:
done